css/styles.css

```diff
@@ -0,0 +1,589 @@
+
+.rabbit-logo{
+    position: relative;
+    left: 218px;
+    top: 3px;
+}
+
+.rabbit-text{
+    position: relative;
+    left: 360px;
+    top: -122px;
+}
+
+
+@font-face {
+  font-family: bebas;
+    src: url('/fonts/bebas.ttf');
+}
+
+@font-face {
+    font-family: ST;
+    src: url('/fonts/StMarie-Thin.otf');
+}
+
+.rabbit-text-span-calibri{
+    font-family: Calibri;
+    color:  #ffb540;
+    font-size: 29pt;
+    letter-spacing: normal;
+}
+
+.rabbit-text-span-bebas{
+    font-family: bebas;
+    color: dimgray;
+    font-size: 27pt;
+    letter-spacing: 2px;
+}
+
+header{
+    background-color: white;
+    top: 2px;
+    z-index: 2;
+    position: fixed;
+    width: 1600px;
+    height: 125px;
+    overflow: hidden;
+    left: 152px;
+}
+
+.home-figure{
+    left: -184px;
+    top: 6px;
+}
+
+.home-span{
+    top: 5px;
+}
+
+.about-image{
+    left: 19px;
+    position: relative;
+}
+
+.about-figure{
+    top: -90px;
+    left: -60px;
+}
+
+.about-span{
+    position: relative;
+    top: 6px;
+    right: 0px;
+}
+
+.work-figure{
+    top: -188px;
+    left: 60px;
+}
+
+.work-span{
+    position: relative;
+    top: 5px;
+    left: 3px;
+}
+
+.blog-span{
+    top: 6px;
+    left: 9px;
+}
+
+.blog-image{
+    position: relative;
+    left: 10px;
+}
+
+.blog-figure{
+    top: -284px;
+    left: 179px;
+}
+
+.contact-span{
+    top: 15px;
+    left: 5px;
+}
+
+.contact-figure{
+    top: -386px;
+    width: 80px;
+    left: 307px;
+}
+
+.contact-image{
+    position: relative;
+    left: 17px;
+    top: 9px;
+}
+
+nav{
+    position: relative;
+    left: 974px;
+    top: -240px;
+    height: 100px;
+    width: 600px;
+}
+
+nav > figure > a{
+     position: relative;
+     top: 1px;
+     text-decoration: none;
+ }
+
+
+a > span{
+    font-family: Calibri;
+    color: darkgray;
+    font-size: 17pt;
+    position: relative;
+    bottom: -31px;
+}
+
+main{
+    left: 142px;
+    top: 103px;
+    position: relative;
+    width: fit-content;
+    height: 1688px;
+    padding: 10px;
+}
+
```

Comment:
Could you please follow style guide and add additional space before { 
selector {
    style: value;
}